task/data/linked_claim.graphql

```diff
@@ -1,20 +1,23 @@
-type LinkedClaim @createModel( 
-     accountRelation: LIST, 
-     description: "Claim or attestation, possibly from 3rd party sources" ) 
-{ 
-    subject: String @string(minLength: 1, maxLength: 256)
-    claim: String @string(minLength: 1, maxLength: 1024)     
-    amount: Int!
```

Comment:
does the ! mean its required?  can we not require Amount in the model, since we might want to use it for other things than impact credits?